test-case/check-userspace-cardinfo.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+##
+## Case Name: check-userspace-cardinfo
+## Preconditions:
+##    N/A
+## Description:
+##    Get the card name from pactl info
+##    check if there is an available card to do the following tests
+## Case step:
+##    1. run pactl to get the cards info
+##    2. check the card name
+## Expect result:
+##    There is at least one available card for test
+##
+
+# shellcheck source=case-lib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
+
+# check pulseaudio runs properly or not
+if ! func_lib_check_pa; then
+    die "Please check whether pulseaudio runs correctly or not"
+fi
+
+func_opt_parse_option "$@"
+
+OLDIFS=$IFS
+dlogc "pactl list cards short"
+cardlist=$(pactl list cards short)
+((available_card=0))
+IFS=$'\n'
+for card in $cardlist; do
+    # pactl list cards short format should be like:
+    # 0	alsa_card.pci-0000_00_1f.3-platform-sof_sdw	module-alsa-card.c
+    dlogi "found card: $(echo "$card" | awk '{print $2}')"
+    usbcard=$(echo "$card" |grep "[Uu][Ss][Bb]")
+    if [ -z "$usbcard" ]; then
+        ((available_card++))
+    fi
+done
+IFS=$OLDIFS
+
+if [ "$available_card" == "0" ]; then
+    # TODO: do more check to give hint why there is no available card
+    die "no available card for test" # There should be at least one card available
```

Comment:
The comment is just repeating the error message.

Reply:
OK, I will remove it